### PR TITLE
Container Instance: Better Warning and Table Output Fix

### PR DESCRIPTION
**Pull request by samkreter**
* Add warning message when trying to create a container instance using an image that does not have a long running process.

* Fix the table output for az container list when the instance view state is null.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/command_modules/azure-cli-container/azure/cli/command_modules/container/_validators.py [hunk 1]
```diff
@@ -37,6 +42,13 @@ def validate_gitrepo_directory(ns):
         raise CLIError("The git repo directory cannot contain '..'")
 
 
+def validate_image(ns):
+    if ns.image.split(':')[0] in short_running_images and not ns.command_line:
+        logger.warning('Image "%s" has no long running process. '
+                       'A command is required for the container to stay running.',
+                       ns.image)
```
Comment (tjprescott) [inline src/command_modules/azure-cli-container/azure/cli/command_modules/container/_validators.py]: What does this mean?  To me it sounds like some creates are LRO and some aren't... is that the case?  What command is "required for the container to stay running"?

File: src/command_modules/azure-cli-container/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+0.3.6
++++++
+* Show warning when creating a container group with an image without a long running process
+* Fix table output issues
```
Comment (tjprescott) [inline src/command_modules/azure-cli-container/HISTORY.rst]: Better to say which commands were affected and now are fixed.

File: src/command_modules/azure-cli-container/azure/cli/command_modules/container/_validators.py [hunk 3]
```diff
@@ -37,6 +42,14 @@ def validate_gitrepo_directory(ns):
         raise CLIError("The git repo directory cannot contain '..'")
 
 
+def validate_image(ns):
+    if ns.image.split(':')[0] in short_running_images and not ns.command_line:
+        logger.warning('Image "%s" has no long running process. The "--command-line" argument must be used to start a '
+                       'long running process inside the container for the container group to stay running. '
+                       'Ex: /bin/bash',
```
Comment (yangl900) [inline src/command_modules/azure-cli-container/azure/cli/command_modules/container/_validators.py]: could you get an aka.ms link from @jluk  that point to our documentation explain this in details?

Comment (jluk) [reply src/command_modules/azure-cli-container/azure/cli/command_modules/container/_validators.py]: aka.ms/aci/troubleshoot points to our general doc, I can add in another header about this.

**Pull request merged**